README.md

```diff
@@ -1,10 +1,22 @@
-# Awesome Memento [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+# Awesome Memento [![Awesome](https://awesome.re/badge.svg)](https://github.com/sindresorhus/awesome)
 My list of things related to software, literature, and other content for Memento ([RFC7089](https://tools.ietf.org/html/rfc7089)).
```

Comment:
The term "My list" is a little too personal, awesome lists are often community resources.

Reply:
@ibnesayeed, I agree on "My" and provided an update in fe4a713. If you can point me to a repo that has this setup already, I can model a configuration for CI/CD on this repo.